protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+It is reasonable to assume that `quantum` will be set to a value around `1 USD` (though there will likely be quite significant variation from this for assets that are not well correlated with USD).
+Therefore, for example, to reward futures markets when they reach a lifetime traded notional over 1 mil USD, then this parameter should be set to around `1000000`. Any decimal value strictly greater than `0` is valid. 
+
+
+## Reward process high level
+
+At a high level, rewards work as follows:
+
+- Reward metrics are calculated for each combination of [reward type, party, market].
+ The calculation used for the reward metric is specific to each reward type.
+-  A transfer is made to the reward account(s) for a specific reward type, for one or more markets. This is either a one-off transfer for a single market, or a recurring transfer for one, several, or all markets. (See [transfers](./0057-TRAN-transfers.md).)
+- At the end of the epoch, the entire balance of each reward account is distributed to the parties with a non-zero reward metric pro-rata by their reward metric.
+If the sum of all reward metrics is zero, the balance rolls over to the next epoch.
+
+
 ## Reward metrics
 
-Reward metrics need to be calculated for every relevant Vega [party](0017-PART-party.md). By relevant we mean that the metric is `> 0`. 
+Reward metrics are scoped by [reward type, market, party] (this triplet can be thought of as a primary key for reward metrics).
+Therefore a party may be in scope for the same reward type multiple times but no more than once per market.
+Metrics will be calculated at the end of every epoch, for every eligible party, in each market for each reward type.
+Metrics only need to be calculated where the [market, reward type] reward account has a non-zero balance of at least one asset. 
+
+
+### Market activity (fee based) reward metrics
+
+There will be three market activity reward metrics calculated based on fees (as a proxy for activity). 
+Each of these represents a reward type with its own segregated reward accounts for each market.
+
+1. Sum of maker fees paid by the party on the market this epoch
+1. Sum of maker fees received by the party on the market this epoch
+1. Sum of LP fees received by the party on the market this epoch
+
+Theese metrics apply only to the sum of fees for the epoch in question.
+That is, the metrics are reset to zero for all parties at the end of the epoch.
+If the reward account balance is 0 at the end of the epoch for a given market, any parties with non-zero metrics will not be rewarded for that epoch and their metric scores do not roll over (they are still zeroed).
+
+Market activity (fee based) reward metrics are not stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and are reset after a checkpoint restart.
```

Comment:
that's not how it's implemented - as discussed [here](https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1660744259282999?thread_ts=1660727163.794979&cid=CAHA5EX0F) (although in the context of market bonus) they are actually stored in the checkpoint for all metrics and are continued after the checkpoint from that point. Not sure what the desired behaviour is.

Reply:
This is discussing the fee based metrics (e.g. cumulative maker fees paid this epoch) - my understanding was that these are not stored but that the market's cumulative trade value for fee purposes and the flags for market creator rewards paid would be in LNL checkpoints.

I think that understanding is in line with the linked slack convo, but apologise if I'm missing something - let me know if so.

Reply:
well tbf in the chat we only discussed the market creation aspect, but in the checkpoint we store the whole party contribution to fees in the market, so in case we take a checkpoint close to the end of the epoch, when we restore, they don't lose reward (if there's balance) at the end of the epoch.